### PR TITLE
Add aria-label / aria-lablledby props to Modal

### PR DESCRIPTION
**Pull request by pcln-barry**
Resolves https://github.com/priceline/design-system/issues/768

Looks like the pre-commit hook linted a few things, I can revert that if it was incorrect.

### CODE REVIEW

File: packages/modal/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-modal",
-  "version": "3.3.0",
+  "version": "3.4.0",
```
Comment (craigpalermo) [inline packages/modal/package.json:3]: Should we hold off on version bumps until the hackathon is over?

**Comment by craigpalermo** [comment]
LGTM pending a decision on bumping the version as part of this PR

File: packages/modal/package.json (hunk 1 above)
Comment (pcln-james) [reply packages/modal/package.json:3]: No I think it's fine unless it's a major 😛

Comment (pcln-james) [reply packages/modal/package.json:3]: Unless we want to group all our bumps into one

Comment (craigpalermo) [reply packages/modal/package.json:3]: I guess we also don't need to publish each time we bump the version

**Pull request merged**